packages/transport-tcp/test/connection-time.spec.js

```diff
@@ -0,0 +1,87 @@
+/**
```

Comment:
No need to add `packageDocumentation` tags - test files are not part of the generated docs and besides, you should only have one `packageDocumentation` tag per module - the one for `@libp2p/tcp` is in the `/src/index.ts` file.